internal/sbi/api_oam.go

```diff
@@ -0,0 +1,33 @@
+package sbi
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+)
+
+func (s *Server) setCorsHeader(c *gin.Context) {
+	c.Writer.Header().Set("Access-Control-Allow-Origin", "*")
+	c.Writer.Header().Set("Access-Control-Allow-Credentials", "true")
+	c.Writer.Header().Set(
+		"Access-Control-Allow-Headers", "Content-Type, Content-Length, Accept-Encoding, "+
+			"X-CSRF-Token, Authorization, accept, origin, Cache-Control, X-Requested-With")
+	c.Writer.Header().Set("Access-Control-Allow-Methods", "POST, OPTIONS, GET, PUT, PATCH, DELETE")
+}
```

Comment:
@ianchen0119 @andy89923 Not sure if it'd be better to use [the official cors middleware](https://github.com/gin-contrib/cors)?

BTW, turn these values into configurable variables would be much more resilient IMO.

Reply:
I suggest adding **TODO** comments and adjust in another PR.